examples/tests/igraph_eulerian_paths.c

```diff
@@ -0,0 +1,79 @@
+#include <stdio.h>
+#include "test_utilities.inc"
+
+int main() {
+
+    igraph_t graph, graph2, graph3, graph4, graph5, graph6, graph7, graph8, graph9;
+    igraph_vector_t res, res2, res3, res4, res5, res6, res7, res8, res9;
+
+    igraph_vector_init(&res, 0);
+    igraph_vector_init(&res2, 0);  
+    igraph_vector_init(&res3, 0);  
+    igraph_vector_init(&res4, 0);
+    igraph_vector_init(&res5, 0);
+    igraph_vector_init(&res6, 0);  
+    igraph_vector_init(&res7, 0);  
+    igraph_vector_init(&res8, 0);     
+    igraph_vector_init(&res9, 0); 
+
+    igraph_small(&graph, 0, IGRAPH_UNDIRECTED, 0,1 , 1,2, -1);
+    igraph_eulerian_paths(&graph, &res);
+    print_vector(&res, stdout);
```

Comment:
Use `print_vector_round` when the elements of the vector are integer valued. You can look in `test_utilities.inc` to see what is available.